devel/balena-cli/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                balena-cli
```

Comment:
One last change, and my apologies for not noticing sooner.  Since you are pulling from Github, it makes the most sense for this port to use the `github` portgroup.

Adding this portgroup would look like the following:
```
PortSystem          1.0
PortGroup           github 1.0

github.setup        balena-io balena-cli 12.55.2 v
github.tarball_from releases
revision            0

categories ...
```

The portgroup provides `github.setup`, which replaces and allows you to drop `name`, `version` and `master_sites`.

I think once you add the `github` portgroup and CI looks good, we should be good to merge.

Reply:
Done.

I had it like this before but `github.tarball_from releases` was the magic I was missing.

Thanks so much for all the help @herbygillot

Reply:
No worries @dylanbr, thanks for this submission